### ISSUE TITLE
Document installation on custom bridge

### ISSUE THREAD

**Issue by unsystemizer** [CounterpartyXCP/federatednode#298]
This is documented in Docker docs, but there may be some tricks so we should test this and potentially document in FedNode documentation.

Possible reasons for using custom bridges would be when there's existing blockchain on another host, or when the host runs other services that conflict with FedNode services (there's already an option to modify mongodb interface, but it doesn't seem documented in FedNode documentation). 

While it's relatively easy to change Docker bridge configuration, it isn't straightforward. For example outgoing connections work, but incoming require additional changes to iptables. Also DNS configuration must continue to work.

**Comment by robby-dermody** [comment]
PR documenting the mongodb interface would be good. I'm hesitant to add a ton to the docs to cover these edge cases, as it will make them harder to parse. A add on section (or better yet, a separate document) detailing fednode gotchas might work though

**Comment by unsystemizer** [comment]
Yes I was thinking the same - we don't want too many options here, this is supposed to be works anywhere & easy. 

I'll leave this open as a to-do for docs and reference updates here as I make progress with the docs.
